tests/sys/progress_bar/tests/01-run.py

```diff
@@ -18,14 +18,14 @@
 
 
 def testfunc(child):
-    for i in range(0, 100, 10):
+    for i in range(0, 101, 10):
```

Comment:
Can you factor this out into its own commit, please? This seems to be unrelated to the Kconfig fix.

Reply:
Sure, done